conf.d/main

```diff
@@ -41,6 +50,15 @@ $SRC/populate-db.exp
 mysql --batch --execute "update faveo.users set user_name='$ADMIN_USER', first_name='$ADMIN_FNAME', last_name='$ADMIN_LNAME' where id='1';"
 /usr/lib/inithooks/bin/faveo-helpdesk.py --email="$ADMIN_MAIL" --pass="$ADMIN_PASS"
 
+#Ensuring queue.php file is configured correctly before artisan commands run
+echo "QUEUE_CONNECTION=redis" >> $CONF
+rm -rf $WEBROOT/bootstrap/cache/*.php
+
+#Disabling bugsnag debbuging service
```

Comment:
It'd be good to explicitly note why we're disabling bugsnag, i.e. because it causes errors during install (and likely when running too).

Reply:
I added lines to the changelog about disabling ioncube loader because it causes errors at build time.